src/msg/async/Event.cc

```diff
@@ -126,10 +126,8 @@ EventCenter::~EventCenter()
   }
   assert(time_events.empty());
 
-  if (notify_receive_fd >= 0) {
-    delete_file_event(notify_receive_fd, EVENT_READABLE);
+  if (notify_receive_fd >= 0)
```

Comment:
could you explain why there is no need to delete `delete_file_event()` in the dtor?